ci-operator/step-registry/optional-operators/cvp-common/scorecard-test/optional-operators-cvp-scorecard-test-ref.yaml

```diff
@@ -0,0 +1,14 @@
+ref:
+  as: optional-operators-cvp-scorecard-test
+  from: cli
```

Comment:
I'm guessing we want to have the `cvp-operator-scorecard` here.

Reply:
Yes, forgot about this one. Thanks for noticing.